tmk_core/common/action.c

```diff
@@ -311,11 +313,35 @@ void process_action(keyrecord_t *record, action_t action)
         /* Mouse key */
         case ACT_MOUSEKEY:
             if (event.pressed) {
-                mousekey_on(action.key.code);
-                mousekey_send();
```

Comment:
@jackhumbert I suspect this change may have messed up `KC_MS_BTN1` and `KC_MS_BTN2`. At least for me, currently these don't work. Do they work for you?